tests/nnstreamer_query/runTest.sh

```diff
@@ -20,7 +20,8 @@ testInit $1
 
 PATH_TO_PLUGIN="../../build"
 PERFORMANCE=0
-TIMEOUT_SEC=5
+SLEEPTIME_SEC=5
```

Comment:
I recommend to please add a comment here about setting this as 5 sec or link the issue number of future reference.

Reply:
I added the comment. Thanks.